modules/ipi-install-configuring-nodes.adoc

```diff
@@ -61,16 +62,22 @@ NICx is a routable network (`baremetal`) that is used for the installation of th
 endif::[]
 
 ifeval::[{product-version} > 4.6]
-.Configuring nodes for Secure Boot
+[id="configuring-nodes-for-secure-boot_{context}"]
+.Configuring nodes for Secure Boot manually
+
+Secure Boot prevents a node from booting unless it verifies the node is using only trusted software, such as UEFI firmware drivers, EFI applications and the operating system.
 
-Secure Boot prevents a node from booting unless it verifies the node is using only trusted software, such as UEFI firmware drivers, EFI applications and the operating system. Red Hat only supports Secure Boot when deploying with RedFish Virtual Media.
+[NOTE]
+====
+Red Hat only supports manually configured Secure Boot when deploying with Redfish virtual media.
+====
 
-To enable Secure Boot, refer to the hardware guide for the node. To enable Secure Boot, execute the following:
+To enable Secure Boot manually, refer to the hardware guide for the node and execute the following:
 
 . Boot the node and enter the BIOS menu.
```

Comment:
Do these need to be sequentially? If yes, change to ordered list

Reply:
That actually renders as an ordered list. You don't have to put the numbers in.